active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
```

Comment:
```suggestion
- Build and strengthen relationships between Django and funding sources
```

Seems to me like we might as well not just think of donors here?

Reply:
I tend to agree - for example I'd like to think about more broadly going after grants (eg I think we have a good shot at raising money from the [Sovereign Tech Fund](https://www.sovereigntechfund.de/) so thinking more broadly than "donations" seems great to me